src/Framework/OpenXmlBasePart.cs

```diff
@@ -81,6 +81,11 @@ internal void Load(OpenXmlPackage openXmlPackage, OpenXmlPart parent, Uri uriTar
             this._uri = uriTarget;
 
             // TODO: should we delay load?
+            if (!this.OpenXmlPackage.Package.PartExists(uriTarget))
+            {
+                throw new OpenXmlPackageException(string.Format("The part with URI target {0} does not exist in the package.", uriTarget));
+            }
```

Comment:
@a-h, I've never run into a situation where a part did not exist. Therefore, the extra invocation of `PartExists()` is overhead that we might want to save.

I assume `GetPart()` throws the exception with the unhelpful message. Therefore, there would be two approaches in my view:

1. Wrap `GetPart()` in a try-catch statement and throw your exception.
2. Have `GetPart()` throw a more verbose exception.

Given that the Open XML SDK now uses `System.IO.Packaging` developed by @EricWhiteDev, it would be possible to do the right thing right there.

@EricWhiteDev, what do you think?